model/src/pyrenew/convolve.py

```diff
@@ -14,33 +14,70 @@
 from __future__ import annotations
 
 from typing import Callable
-
+from pyrenew.transformation import IdentityTransform
 import jax.numpy as jnp
 from jax.typing import ArrayLike
 
 
-def new_convolve_scanner(discrete_dist_flipped: ArrayLike) -> Callable:
+def new_convolve_scanner(discrete_dist_flipped: ArrayLike,
+                         transform: Callable = None) -> Callable:
```

Comment:
Why default to None instead of identity?

Reply:
The [mutable default argument](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) gotcha. In many use cases, it isn't a problem (and very likely isn't here), but I have just gotten in the habit of always using this pattern when a default value is of a mutable type.

```python
if arg is None:
   arg = mutable_default()
```